### ISSUE TITLE
TF 1.14 Can't pickle FlagValues

### ISSUE THREAD

**Issue by grwlf** [TellinaTool/nl2bash#17]
Just found an error during training. Probably, new verison of TensorFlow doesn't allow to save flags using pickle.

```
...
step 4 learning rate 0.0001 dev_perplexity 6.02
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/workspace/mironov/nl2bash/encoder_decoder/translate.py", line 372, in <module>
    tf.app.run()
  File "/usr/local/lib/python3.6/dist-packages/tensorflow/python/platform/app.py", line 40, in run
    _run(main=main, argv=argv, flags_parser=_parse_flags_tolerate_undef)
  File "/usr/local/lib/python3.6/dist-packages/absl/app.py", line 300, in run
    _run_main(main, args)
  File "/usr/local/lib/python3.6/dist-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/workspace/mironov/nl2bash/encoder_decoder/translate.py", line 360, in main
    save_hyperparameters()
  File "/workspace/mironov/nl2bash/encoder_decoder/translate.py", line 196, in save_hyperparameters
    pickle.dump(FLAGS, o_f)
  File "/usr/local/lib/python3.6/dist-packages/absl/flags/_flagvalues.py", line 640, in __getstate__
    raise TypeError("can't pickle FlagValues")
TypeError: can't pickle FlagValues
Makefile:41: recipe for target 'train' failed
make: *** [train] Error 1
```

**Comment by grwlf** [comment]
One dumb workaround is to just comment-out saving of hyperparameters

**Comment by todpole3** [comment]
This has been fixed with the recent Tensorflow 2.0 update.

Thanks for pointing it out.
